save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/ExecutionView.kt

```diff
@@ -88,44 +82,49 @@ external interface StatusProps<D : Any> : TableProps<D> {
     var status: TestResultStatus?
 
     /**
-     * Name of test suite
+     * File name to filter by
+     */
+    var testName: String?
```

Comment:
Still didn't change `testName` to `testFileName` :c

Reply:
Done